tests/notebooks/training/training-integration.ipynb

```diff
@@ -430,7 +433,7 @@
     "container = V1Container(\n",
     "    name=PYTORCHJOB_CONTAINER,\n",
     "    image=PYTORCHJOB_IMAGE,\n",
-    "    args=[\"--backend\", \"gloo\"],\n",
+    "    args=[\"--backend\", \"gloo\", \"--epochs\", \"2\"],\n",
```

Comment:
suggestion: comment that this is modified from the upstream example and why (to reduce the test time?)